vendor/github.com/Microsoft/hcsshim/hcn/hcnpolicy.go

```diff
@@ -76,10 +76,10 @@ type SubnetPolicy struct {
 // NatFlags are flags for portmappings.
 type NatFlags uint32
 
-const (
-	NatFlagsNone NatFlags = iota
-	NatFlagsLocalRoutedVip
-	NatFlagsIPv6
+var (
```

Comment:
We shouldn't be making changes in the vendor repo. We should be making the required changes in https://github.com/microsoft/hcsshim and creating a tag when the changes are merged, then revendor hcsshim in this directory. 
This will ensure the changes aren't wiped out in the future due to any version upgrade.

https://github.com/microsoft/windows-container-networking/blob/0f859017f2803422304bd65f3e974789b23e0fca/go.mod#L7

Ref: https://git-scm.com/book/en/v2/Git-Basics-Tagging

Reply:
Yes, the vendor changes were temporarily made and pushed for validation. Local tests were failing due to infra issues. Thanks for the comment!